Running-Mastodon/Production-guide.md

```diff
@@ -48,6 +48,9 @@ server {
 
 
   add_header Strict-Transport-Security "max-age=31536000; includeSubDomains";
+  # You MUST replace `example.com` by your server name.
```

Comment:
also as I said, it would be nice having a warning here *and* down in the doc, around where the reader is instructed on installing Postgres maybe?